apps/src/templates/lessonOverview/LessonStandards.jsx

```diff
@@ -44,7 +42,7 @@ const expandModeShape = PropTypes.oneOf(
 
   // This component and all its descendants should be expanded.
   ExpandMode.ALL
-);
+]);
```

Comment:
nice catch! Odd that this wasn't giving a proptype validation error.

Reply:
It was showing one in the console and when I ran the tests but wasn't preventing the page from loading.